R/utils-classes.R

```diff
@@ -316,7 +315,10 @@ show_query.bcdc_promise <- function(x, ...) {
 #' }
 show_query.bcdc_sf <- function(x, ...) {
 
-  url <- url_format(attributes(x)$url)
+  url <- attributes(x)$url
+  query_list <- attributes(x)$query_list
+  cql_filter <- query_list$CQL_FILTER
+  query_list$CQL_FILTER <- NULL
```

Comment:
Why do we set `query_list$CQL_FILTER` to `NULL` here? It's not used again is it?

Reply:
So what happened here is that when showing the body, these huge wkt polygons really bonked the console when printing. So I separate then out into `query_list` and `CQL_FILTER`. So when I printed we print the query_list it omits the `CQL_FILTER` then we add a truncated `CQL_FILTER`.